### DISCUSSION TITLE
too many files of oap-server.log without deletion automatically

### DISCUSSION THREAD

**Discussion by AngryMills** [apache/skywalking#7725] [Q&A]
oap-server runs online for 2 months,and make 81G logs.
`<DefaultRolloverStrategy>` of log4j2.xml is not efficient.
Can add a default config for maximum retention days?
![image](https://user-images.githubusercontent.com/19707438/133577626-de531a66-94c7-437b-8225-e02b765054ff.png)

**Comment by wu-sheng** [comment]
This is log4j mechanism, out of SkyWalking scope. We don't change anything from standard log framework.
